src/DDTrace/Util/Runtime.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace DDTrace\Util;
+
+/**
+ * Utilities related tot he PHP runtime
+ */
+class Runtime
```

Comment:
Just to lock this API down a bit. :)
```suggestion
final class Runtime
```